index.html

```diff
@@ -144,28 +144,34 @@ <h2>Introduction</h2>
       <h2>Identifiers</h2>
       <em>This section is non-normative.</em>
       <p>
-          When expressing statements about a specific entity, such as a person, product, or organization,
-          it is often useful to use some kind of identifier so that others can express statements about
-          the same thing. You may have noticed that the VC Data Model specification contains numerous
-          examples where the choice of identifier for the subject of the credential (the thing the claims
-          are about) has been a <a href="https://w3c-ccg.github.io/did-primer/">decentralized identifier</a>,
-          also known as a DID. An example of a DID is <code>did:example:123456abcdef</code>.
+When expressing statements about a specific entity, such as a person, product,
+or organization, it is often useful to have an identifier for it so that others
+can express statements about the same thing. The <a>verifiable credentials</a>
+<a href="https://w3c.github.io/vc-data-model/">data model specification</a>
```

Comment:
`w3c.github.io`-based links should be changed to `w3.org`-based links.

Reply:
Changed in [b1c845b](https://github.com/w3c/vc-imp-guide/pull/27/commits/b1c845b435ab6438c1e6e2eed60e213d4aa2f037)